content/links.md

```diff
@@ -227,13 +227,76 @@ instance_groups:
+
+* **as** [String]: Overrides the name of the provider defined in the release spec. Example:
+```yaml
+instance_groups:
+- name: my_instance_group
+  jobs:
+  - name: my_provider_job
+    provides:
+      bar: {as: new_name}
+```
+* **shared** [Boolean]: *Default is false* Sets whether this provider is consumable from another deployment. See [cross deployment links](#cross-deployment).
+
+Providers that are specified as `nil` will cause your *explicit* consumer to fail resolution.
+
+Deployment Manifest:
+```yaml
+instance_group:
+- name: db_ig
+  jobs:
+  - name: db_job
+    provides:
+      database: nil
+```
+
+##### Consumers
+Explicitly defined consumers can have the following optional properties:
+
+* **from** [String]: Overrides the name of the provider to consume. This should match the name defined in the provider's release spec or the name defined by provider's `as` property in the manifest.
+* **deployment** [String]: The name of the deployment to consume from. If the deployment provided does not exist, it will fail with an error. If the name specified is the same as the current deployment, it will be considered non-cross-deployment.
+* **network** [String]: The name of a network used by the provider. See [custom network linking](#custom-network).
+* **ip_addresses** [Boolean]: *Default is false* Boolean for preferring usage of ip addresses instead of DNS names. In cases such as dynamic network, ip addresses are not available and will still return DNS names. See [dns](dns#links) for more details.
+
+Optional consumers may be defined as `nil` in your deployment manifest to block linking.
+
+Deployment Manifest:
+```yaml
+instance_groups:
+- name: app_ig
+  jobs:
+  - name: app
+    consumes:
+      database: nil
+```
+
 ### Implicit linking {: #implicit }
+Implicit linking is defined as not having your consumer in your deployment manifest but only lives in your release specification.
 
-If a link type is provided by only one job within a deployment, all release jobs in that deployment that consume links of that type will be implicitly connected to that provider.
+If a link type is provided by only one job within a deployment, all release jobs in that deployment that implicitly consume links of that type will resolve to that provider.
 
-Optional links are also implicitly connected; however, if no provider can be found, they continue to be `nil`.
+Providers that are specified as `nil` will cause your *implicit* consumer to fail resolution. Unless your consumer is defined as `optional` in your release.
 
-Implicit linking does not happen across deployments.
+Deployment Manifest:
+```yaml
+instance_groups:
+- name: app_ig
+  jobs:
+  - name: db_job
+    provides:
+      database: nil
+  - name: app
+```
+
+Optional links that can not be satisfied implicitly will return `nil` in the template rendering.
+
+Implicit linking does *not* happen across deployments.
```

Comment:
> Implicit linking does *not* happen across deployments.

Maybe: "Implicit linking is not supported across deployments." ?

Reply:
Will update.